src/EditorFeatures/CSharpTest/Classification/SemanticClassifierTests.cs

```diff
@@ -4439,5 +4439,17 @@ public async Task BasicRecordStructClassification(TestHost testHost)
                 testHost,
                 RecordStruct("R"));
         }
+
+        [Theory]
+        [CombinatorialData]
+        public async Task BasicFileScopedNamespaceClassification(TestHost testHost)
+        {
+            await TestAsync(
+@"namespace NS;
+
+class C { }",
+                testHost,
+                Namespace("NS"));
```

Comment:
would be good to put this (or a copy) in the TotalClassifierTests.  Then we can validate that the rest of hte tokens classify properly.

Reply:
Done